util/test/rdtest/capture.py

```diff
@@ -183,6 +206,15 @@ def run_and_capture(exe: str, cmdline: str, frame: int, *, frame_count=1, captur
     control.run(keep_running=lambda x: len(x.captures()) < captures_expected)
 
     captures = control.captures()
+    log.print(f'Retrieved {len(captures)}')
```

Comment:
nit: without context in the string this will only print as `Retrieved 1` etc - could you change this ?
```suggestion
    log.print(f'Retrieved {len(captures)} captures')
```